com.chartboost.mediation/Editor/Adapters/Constants.cs

```diff
@@ -0,0 +1,34 @@
+using System.IO;
+using UnityEngine;
+
+namespace Chartboost.Editor.Adapters
+{
+    public class Constants
+    {
+        public const string Unselected = "Unselected";
+        public const string ChartboostMediationPackageName = "com.chartboost.mediation";
+        public const string AndroidVersionInMainTemplate = "%ANDROID_VERSION%";
+        public const string IOSVersionInMainTemplate = "%IOS_VERSION%";
+        
+        public const string PathToMainTemplate = "Packages/com.chartboost.mediation/Editor/Adapters/Templates/TemplateChartboostMediation.xml";
+        public const string PathToAdapterTemplate = "Packages/com.chartboost.mediation/Editor/Adapters/Templates/TemplateAdapter.xml";
+
+        public static readonly string PathToPackageGeneratedFiles = Path.Combine(Application.dataPath, "com.chartboost.mediation");
+        public static readonly string PathToEditorInGeneratedFiles = Path.Combine(PathToPackageGeneratedFiles, "Editor");
+        public static readonly string PathToAdaptersDirectory = Path.Combine(PathToEditorInGeneratedFiles, "Adapters");
+        public static readonly string PathToMainDependency = Path.Combine(PathToEditorInGeneratedFiles, "ChartboostMediationDependencies.xml");
+        public static readonly string PathToSelectionsFile = Path.Combine(PathToEditorInGeneratedFiles, "selections.json");
+        
+        // Quirky networks
```

Comment:
Let's have a proper comment here. For example, a reason why they are being used slightly different than the others.